brainstormforce/astra
# fix: Woo/EDD cart flyout opens on Desktop & Responsive color options not working for both.

### Description
 - Trello task - https://trello.com/c/3YndIyFF/1202-woo-and-edd-cart-tray-option-not-working-on-backend-and-frontend-except-separator-https-dpr-i-t8w8c6

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - Check all the responsive color options for Woo/EDD cart in the frontend & backend.- https://d.pr/i/T8w8c6
 - Check that the Cart flyout should only open on Mobile & tablet & not on desktop.

### Checklist:
